src/components/shared/RadioField/index.tsx

```diff
@@ -60,14 +60,15 @@ export const RadioField = ({
 type RadioGroupProps = {
   children: React.ReactNode | React.ReactNodeArray;
   inline?: boolean;
-};
+} & JSX.IntrinsicElements['div'];
```

Comment:
This is to be able to pass all the `aria*` elements and spread them. It'd be way too many props to specific individually.